documentation/tutorial.md

```diff
@@ -68,3 +78,660 @@ tutorial overlord    localhost/localhost  2.9.37   unsupported  23:20:53Z
 Model "admin/tutorial" is empty.
 ```
 
+
+## Deploy Charmed MongoDB
+To deploy Charmed MongoDB, fetch the charm from [Charmhub](https://charmhub.io/mongodb?channel=dpe/edge) and deploy it to your model:
+```shell
+juju deploy mongodb --channel dpe/edge
+```
+
+Juju will now fetch Charmed MongoDB and begin deploying it to the LXD cloud. This process can take several minutes depending on how provisioned your machine is. You can track the progress by running:
+```shell
+watch -c juju status --color
+```
+
+This will show charmed MongoDB along with its current status and other helpful information.
+
+Wait until the application is ready, when it is ready `watch -c juju status --color` will show
+```
+Every 2.0s: juju status --color                                                                                         ip-172-31-11-104: Fri Dec  2 11:24:30 2022
+
+Model     Controller  Cloud/Region         Version  SLA          Timestamp
+tutorial  overlord    localhost/localhost  2.9.37   unsupported  11:24:30Z
+
+App      Version  Status  Scale  Charm    Channel   Rev  Exposed  Message
+mongodb           active      1  mongodb  dpe/edge   96  no
+
+Unit        Workload  Agent  Machine  Public address  Ports      Message
+mongodb/0*  active    idle   0        10.23.62.156    27017/tcp
+
+Machine  State    Address       Inst id        Series  AZ  Message
+0        started  10.23.62.156  juju-d35d30-0  focal       Running
+```
+
+## Access MongoDB
+*Disclaimer: this part of the tutorial accesses MongoDB via the `admin` user. **Do not** directly interface with the admin user in a production environment. In a production environment [always create a separate user](https://www.mongodb.com/docs/manual/tutorial/create-users/) and connect to MongoDB with that user instead. Later in the section covering Relations we will cover how to access MongoDB without the admin user.*
+
+For this part of the Tutorial we will access MongoDB - via the MongoDB shell `mongosh`. You can read more about the MongoDB shell [here](https://www.mongodb.com/docs/mongodb-shell/).
+
+
+### Install the MongoDB shell
+While MongoDB is installed within the LXD containers that the application is hosted on, it is not installed on your machine. This means that you will have to install the MongoDB shell yourself. Install `mongosh` by following the official [instructions on how to install the MongoDB shell](https://www.mongodb.com/docs/mongodb-shell/install/#std-label-mdb-shell-install).
+
```

Comment:
In general, I feel as though these instructions on installing mongosh are out of data and possibly unclear. I'm also not sure if I should only be following this first page, or if I have to use any others. I've come back to your tutorial at the end of the first page (unsure if I've actually correctly installed it or not).